src/tests/ui/planner/iteration.ts

```diff
@@ -22,11 +23,15 @@ export class Iteration extends ui.BaseElement {
   month = new ui.Clickable(this.$('.headermonthtxt'), 'month');
   year = new ui.Clickable(this.$('.yearlabel'), 'year');
     
-  async addNewIteration(iterationName: string, parentIteration: string ) {
+  async addNewIteration(iterationName: string, parentIteration?: string, withDates?: boolean) {
     await this.iterationName.enterText(iterationName);
-    await this.parentIteration.enterText(parentIteration);
-    await this.parentIteration.clickWhenReady();
-    await this.selectCalendarDate();    
+    if(parentIteration) {
+      await this.parentIteration.enterText(parentIteration);
+      await this.parentIteration.clickWhenReady();
```

Comment:
@jarifibrahim change await this.parentIteration.clickWhenReady(); to await this.parentIteration.select(parentIteration)

Reply:
Done